packages/barqode/src/lib/components/barqode-stream.svelte

```diff
@@ -26,7 +26,7 @@
 	let isMounted = $state(false);
 
 	// derived camera settings based on props and state
-	let cameraSettings = $derived({
+	const cameraSettings = $derived({
```

Comment:
just curious, why `const` over `let` for this?

I have not learned the pattern yet. in the [svelte 5 docs](https://svelte.dev/docs/svelte/$derived) they use `const` in some places but `let` in (most?) other places.

Reply:
I prefer the pattern of `let`  for variables that can be _reassigned_ whereas `const` for anything that can't be reassigned.

You cannot reassign derived so it makes no sense for them to be `let`s 😃

Reply:
And on the other hand `$state` can be reassigned, it should be `let`.

Reply:
thanks!